### DISCUSSION TITLE
Missing ScreenStackHeaderSubview in Windows

### DISCUSSION THREAD

**Discussion by KacperPaszkowski** [software-mansion/react-native-screens#1242] [Ideas]
Hi,

There is no ScreenStackHeaderSubview component in windows so it's impossible to make custom header on Windows.

**Comment by kacperkapusciak** [comment]
Hi @KacperPaszkowski,

The current implementation of `react-native-screens` for Windows was just to support `react-navigation`, and the implementation is far from being feature complete.

I've contacted the original authors of the implementation of Screens for Windows and they'll have a look at this after the winter break.

I'm transferring this issue to the discussions because this is a feature request rather than a bug.

Cheers
